types/httpstatus.pp

```diff
@@ -1 +1 @@
-type Stdlib::HttpStatus = Integer[100, 599]
+type Stdlib::HttpStatus = Stdlib::Http::Status
```

Comment:
I wonder if this type can be deprecated.

Reply:
i would also like to deprecate this in favour of `Stdlib::Http::Status` but not sure of the process for that.  i regret not namespacing this when i first introduced it :(

Reply:
can always do this bit in a separate pr if that makes things easier?